### ISSUE TITLE
nameof returns different casing for members than reflection metadata

### ISSUE THREAD

**Issue by DanTup** [bridgedotnet/Bridge#2604]
Deck: https://deck.net/c735b4c6b417f61719c3d78c0f78ad72

Using `nameof()` seems to return incorrect casing for reflection:

```cs
public static bool TestProperty => false;

// ...

type.GetProperty(nameof(TestProperty), BindingFlags.Static)
```

This returns null, however if you change `nameof(TestProperty)` to `"TestProperty"` then everything works fine. The string written for `nameof(TestPRoperty)` is `testProperty` but the reflection metadata has PascalCasing.

**Comment by geoffreymcgill** [comment]
Hi @DanTup – Seems like this might be solved in **master**. Can you confirm the following sample is working as expected?

https://dev.deck.net/b44dc3d791a110350a96e3986b55a165

**Comment by DanTup** [comment]
Yep, that looks good to me! :-)

**Comment by geoffreymcgill** [comment]
I'll close this issue, but we'll also try to find the issue or pr where this was fixed.

**Comment by geoffreymcgill** [comment]
Fixed in #2375.
